CMakeLists.txt

```diff
@@ -4,20 +4,12 @@ project(demonstrator)
 # Print the value of CMAKE_SOURCE_DIR for debugging
 message(STATUS "CMAKE_SOURCE_DIR: ${CMAKE_SOURCE_DIR}")
 
-include(${CMAKE_BINARY_DIR}/conan_toolchain.cmake)
-
 add_executable(demonstrator src/demonstrator.c)
-find_package(module_a REQUIRED)
-find_package(module_b REQUIRED)
+find_package(module_a CONFIG REQUIRED)
+find_package(module_b CONFIG REQUIRED)
 target_link_libraries(demonstrator module_a::module_a module_b::module_b)
 
-# Add include directories from Conan dependencies
-target_include_directories(demonstrator PRIVATE ${CONAN_INCLUDE_DIRS})
```

Comment:
Unnecessary, the CONAN_INCLUDE_DIRS shouldn't be used directly in most cases, the ``module_x::module_x`` targets contain all the information already.

Reply:
this was me going around in circles because i thought my build error was in this repo but it was in fact in the module repos.